### PR TITLE
Nov mailing, to review, missing changelog

### PR DESCRIPTION
**Pull request by atomgalaxy**
Dear Caleb,

if you could review and write the changelog, I'd be much obliged.

G

### CODE REVIEW

File: P2461.md [hunk 1]
```diff
@@ -17,13 +17,16 @@ author:
 
 The attribute-derived syntax for contracts is limiting and steps on the shared
 space between C and C++. This paper explores an alternative syntax that should
-hopefully be more powerul while being able to express the same semantics.
+offer an eaasier extension path.
```
Comment (C-EDG) [inline P2461.md]: Typo - "easier"

File: P2461.md [hunk 2]
```diff
@@ -565,35 +720,10 @@ This syntax kind-of works, but is not proposed, and there is nowhere to specify
...
-expression for the purposes of optimisation, *but they can either all be
-omitted, or none may*, for a given statement, including the closure.
-
-We therefore have a plausible RAII-based metaphor that people already understand.
-
 ## Summary
 
 - The closure-based syntax makes it obvious when values are captured, and even
-  suggests an implementation - just put the closures on the stack before the
+  hints an implementation - just put the closures on the stack before the
```
Comment (C-EDG) [inline P2461.md]: Need "at" between "hints" and "an"

File: P2461.md [hunk 3]
```diff
@@ -605,37 +735,67 @@ We therefore have a plausible RAII-based metaphor that people already understand
...
+| 20 | `T &&`       | `[=]` | `post` |     | y    |
+| 21 | `T const &&` | `[&]` | `pre`  | y   | y    |
+| 22 | `T const &&` | `[&]` | `post` | y   | y    |
+| 23 | `T const &&` | `[=]` | `pre`  |     | y    |
+| 24 | `T const &&` | `[=]` | `post` |     | y    |
+
+- **MVP**: this paper.
+- **Ext**: this paper plus minimal "allow captures" extension.
+- **(!)**: Capturing by value in postconditions makes no sense to the caller,
+  therefore it's not a well-behaved contract and we should diagnose (2).
```
Comment (C-EDG) [inline P2461.md]: Am I misreading something? Isn't (2) capture-by-reference?

I think you mean to say here that capturing a non-const by reference in postconditions makes no sense to the caller. Capturing by value I think is the solution, no?

**Review by C-EDG — APPROVED** [review]
Possible logic error. I'm not sure if I'm misunderstanding things due to my brain fog/fatigue or if you've made the error. Please double-check. If you think you're correct, feel free to disregard the concern.

I like the added examples of future uses, I think they really drive home the flexibility and power of this syntax. All of those examples feel natural and smooth, easy-to-learn, easy-to-read. The attribute forms I think would be a lot uglier. I feel quite proud to be championing this version.

File: P2461.md (hunk 1 above)
Comment (atomgalaxy) [reply P2461.md]: fixed

File: P2461.md (hunk 2 above)
Comment (atomgalaxy) [reply P2461.md]: nice catch, thanks.

File: P2461.md (hunk 3 above)
Comment (atomgalaxy) [reply P2461.md]: Fixed.

**Pull request merged**